src/lib/slide-toggle/slide-toggle.ts

```diff
@@ -65,13 +60,18 @@ export class MdSlideToggle implements AfterContentInit, ControlValueAccessor {
   private _isMousedown: boolean = false;
   private _slideRenderer: SlideToggleRenderer = null;
 
-  @Input() @BooleanFieldValue() disabled: boolean = false;
   @Input() name: string = null;
   @Input() id: string = this._uniqueId;
   @Input() tabIndex: number = 0;
   @Input() ariaLabel: string = null;
   @Input() ariaLabelledby: string = null;
 
+  private _disabled: boolean = false;

```

Comment:
Can we move this up to the other private variables? And also make the `_hasFocus` private while you are at it? :)